tasks/task1.ipynb

```diff
@@ -125,10 +120,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "my_model.T = F.Temperature(500)  # K"
+    "my_model.T = F.Temperature(value=500)"
```

Comment:
is this needed? I thought not having value= would be simpler

Reply:
I figured might be better for the 1st exercise, plus the 'value' argument is mentioned in the describing section so figured it might link a bit better